lib/manager/terragrunt/index.ts

```diff
@@ -0,0 +1,10 @@
+import * as hashicorpVersioning from '../../versioning/hashicorp';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  commitMessageTopic:
+    'Terragrunt {{managerData.terragruntDependencyType}} dependecy {{depNameShort}}',
```

Comment:
```suggestion
    'Terragrunt dependency {{depNameShort}}',
```

Use `depType` if you must, but not `managerData`.

Reply:
Got it!